src/Export/Adapters/ExportItemAdapter.php

```diff
@@ -157,6 +169,12 @@ public function adaptVariant(Item $item, ProductEntity $product): ?Item
             foreach ($this->adapterFactory->getShopwarePropertiesAdapter()->adapt($product) as $property) {
                 $item->addProperty($property);
             }
+
+            $optionAttributes = $this->adapterFactory->getVariantConfigurationAdapter()->adapt($product);
+
+            foreach ($optionAttributes as $attribute) {
+                $item->addMergedAttribute($attribute);
+            }
```

Comment:
With the use of the `adapt` function, this should fit into the max line length

```suggestion
            foreach ($this->adapterFactory->getVariantConfigurationAdapter()->adapt($product) as $attribute) {
                $item->addMergedAttribute($attribute);
            }
```